src/Resources/Resources/help/Get-AzADServicePrincipal.md

```diff
@@ -55,12 +55,6 @@ Get-AzADServicePrincipal [-Select <String[]>] -ServicePrincipalName <String> [-F
  [-AppendSelected] [-DefaultProfile <PSObject>] [<CommonParameters>]
 ```
 
-### List
-```
-Get-AzADServicePrincipal [-Select <String[]>] [-Count] [-First <UInt64>] [-Skip <UInt64>] [-AppendSelected]
- [-DefaultProfile <PSObject>] [<CommonParameters>]
-```
-
```

Comment:
Is this expected?

Reply:
> Is this expected?

Yes this parameter set "List" was incorrect